llvm/include/llvm/DebugInfo/DWARF/DWARFDebugArangeSet.h

```diff
@@ -73,6 +73,12 @@ class DWARFDebugArangeSet {
     return desc_iterator_range(ArangeDescriptors.begin(),
                                ArangeDescriptors.end());
   }
+
+  size_t getDescriptorsSize() const { return ArangeDescriptors.size(); }
+
+  const Descriptor &getDescriptiorRef(uint32_t I) const {
```

Comment:
Should we add an `assert(I < ArangeDescriptors.size())`?

Reply:
I would actually say this shouldn't exist. These are very lldb-like functions, and as far as I can tell, the only place where you're actually using them is in tests -- where we have better options.

Reply:
Removed these two functions.